app/services/caxe.js

```diff
@@ -0,0 +1,8 @@
+import Ember from 'ember';
```

Comment:
The term 'caxe' should either be explained somewhere prominent, or, possibly even better, replaced with something more descriptive (that's not "cakes" ;) )

Reply:
Add a docstring!